ice-rest-catalog/src/main/java/com/altinity/ice/rest/catalog/internal/maintenance/MaintenanceScheduler.java

```diff
@@ -119,6 +130,30 @@ public void performMaintenance() {
 
             table.rewriteManifests().rewriteIf(manifest -> true).commit();
             table.expireSnapshots().expireOlderThan(olderThanMillis).commit();
+
+            if (orphanFileExpirationDays == 0) {
+              logger.info(
+                  "Skipping orphan file removal for table {} since orphanFileExpirationDays config was not set",
```

Comment:
maybe this can be avoided for every table if we have a separate iteration, not sure if this is the best option.